### PR TITLE
Update to use grid system from USWDS

### PR DESCRIPTION
**Pull request by alexiscott**
…swds.

### CODE REVIEW

**Comment by alexiscott** [comment]
Adding a couple of Jest tests.

**Comment by alexiscott** [comment]
On Second thoughts, I'm not sure how useful test would be for these classes. They are pretty arbitrary. 

There is a another piece that I think might warrant it's own ticket: Adding breakpoints as per the the existing site, and ensuring that the container applies those for max-width.

Let me know your thoughts @dgading ? And if there are any stones left unturned here?

File: src/assets/config.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "container": "container",
+  "container": "grid-container",
```
Comment (dgading) [inline src/assets/config.json:2]: I would just remove this line completely. We should move to inline classes to better handle possible one off things.

**Review by dgading — CHANGES_REQUESTED** [review]
If you can just remove the container config and then on each page swap out the dc-page and config.container with the grid-container class. That extra step isn't really saving us anything and just makes it more complicated to possibly change a single page as we could break other pages when messing with config. If we had a lot pages it would make sense, but we only have like 5 pages to deal with.

**Review by dgading — APPROVED** [review]
Looks good!

**Pull request merged**